packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -1372,6 +1476,155 @@ describe('meeting', () => {
       });
     });
 
+    describe('joinMeeting', () => {
+      const dataError = 'Something went wrong...';
+
+      const mockjoinMeetingParams: meeting.JoinMeetingParams = {
+        joinWebUrl: new URL('https://example.com'),
+        source: meeting.EventActionSource.Other,
```

Comment:
These tests look like a big improvement to me, thanks! The only tests I see missing (both framed and frameless) is that you never validate passing a source _except_ for `meeting.EventActionSource.Other`. That will make it possible for bugs to creep in in the future because `Other` is also the default value for `source` in your code. 2 options here:

1) You make the default value of Source here in your mock something than Other (for both framed and frameless tests). Then your `should successfully send the joinMeeting message` tests should test this case.

2) You just add another test that sets the source to a non default value.

Reply:
Made the changes in the e2e tests to check for different source values.